frontend/src/app/components/UpdateList.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classnames from 'classnames';
+import moment from 'moment';
+import LayoutWrapper from './LayoutWrapper';
+import { newsUpdateL10nId } from '../lib/utils';
+
+export function prettyDate(date) {
+  return moment(date).format('MMMM Do, YYYY');
```

Comment:
Hmm, I think we're going to have to tweak this format() to be more i18n friendly